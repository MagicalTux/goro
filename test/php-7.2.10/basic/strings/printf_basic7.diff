--
     int(6)
     
     -- Calling printf() with one arguments --
008- 4294966185
009- int(10)
008+ 1777777777777777775651
009+ int(22)
     
     -- Calling printf() with two arguments --
012- 4294966185 4293732729
013- int(21)
012+ 1777777777777777775651 1777777777777773224571
013+ int(45)
     
     -- Calling printf() with three arguments --
016- 4294966185 4293732729 2345432
017- int(29)
016+ 1777777777777777775651 1777777777777773224571 10744730
017+ int(54)
     ===DONE===
