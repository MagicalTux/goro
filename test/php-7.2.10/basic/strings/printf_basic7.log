
---- EXPECTED OUTPUT
*** Testing printf() : basic functionality - using unsigned format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one arguments --
4294966185
int(10)

-- Calling printf() with two arguments --
4294966185 4293732729
int(21)

-- Calling printf() with three arguments --
4294966185 4293732729 2345432
int(29)
===DONE===
---- ACTUAL OUTPUT
*** Testing printf() : basic functionality - using unsigned format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one arguments --
1777777777777777775651
int(22)

-- Calling printf() with two arguments --
1777777777777777775651 1777777777777773224571
int(45)

-- Calling printf() with three arguments --
1777777777777777775651 1777777777777773224571 10744730
int(54)
===DONE===
---- FAILED
