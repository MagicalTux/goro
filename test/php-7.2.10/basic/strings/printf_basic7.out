*** Testing printf() : basic functionality - using unsigned format ***

-- Calling printf() with no arguments --
format
int(6)

-- Calling printf() with one arguments --
1777777777777777775651
int(22)

-- Calling printf() with two arguments --
1777777777777777775651 1777777777777773224571
int(45)

-- Calling printf() with three arguments --
1777777777777777775651 1777777777777773224571 10744730
int(54)
===DONE===